ci-operator/step-registry/openshift-qe/node-density-cni/openshift-qe-node-density-cni-chain.yaml

```diff
@@ -0,0 +1,11 @@
+
+chain:
+  as: openshift-qe-node-density-cni
+  steps:
+  - ref: openshift-qe-node-density-cni
+  - ref: openshift-qe-write-perfscale-results
+  env:
+  - name: OUTPUT_FILE
+    default: "kube-burner.out"
```

Comment:
This needs to be **kube-burner-ocp.out** to properly process the output file using the sandman script. Currently the sandman script in _openshift-qe-write-perfscale-results_ is set up for both kube-burner and kube-burner-ocp ways of running. To properly set the regex styling of output need to have kube-burner-ocp.out file name